### ISSUE TITLE
wiki standalone instructions for linux host `--config-dir` out of date

### ISSUE THREAD

**Issue by ilovett** [rook/rook#616]
standlone linux host instructions [here](https://github.com/rook/rook/blob/master/Documentation/standalone.md) say to use `--confg-dir /tmp/rook-test` but should actually be `--data-dir`

**Comment by jbw976** [comment]
You are right @ilovett, the documentation as written is incorrect because it's referencing a command line argument that was updated for v0.4 (not yet released) but the instructions are still telling you to download the last release (v0.3.1) that does not have that change yet.

We've enabled continuous publishing for our containers (e.g., `quay.io/rook/rookd:master-latest`) but I'm not sure if we've done the same for our binary archives/zips.  It would be nice if our docs for those could also use a similar concept in use by all the docs that use containers...a "latest" archive/zip instead of a semantic versioned one.  I'll need to look into that.  Thanks for bringing up this issue @ilovett, and I'm glad you've already worked around it.
